app/routes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import logger from './logger';
+import { errorHandler } from './error/errorHandler';
+
+const router = express.Router();
+
+router.get('/healthcheck', async (req, res) => {
```

Comment:
We have a tech debt ticket to change these in our existing FE apps to be in line with the the BE apps.
We might as well get this right over here
https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_queries/edit/5129/?triage=true